File: Swift-C++-IR-Comparison/test/arithmetic-operators/Addition/Swift/UnaryPlus.swift
@main
struct MyMain {

  static func main() {
    let b = -1
    let a = unaryPlus(x: b)
  }

  // Documentation states the following
  // The unary plus operator (+) simply returns the value it operates on, without any change
  static func unaryPlus(x: Int) -> Int {
    let tmp = +x
    return tmp
  }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/UnaryPlus.cpp

struct MyMain
{
    static int main() 
    {
        return unaryPlus(x: -1);
    }

    // Documentation states the following
    // The unary plus operator (+) simply returns the value it operates on, without any change
    static int unaryPlus (int x)
    {
        int a = x;
        int b = +a;
        return b;
    }

};

int main()
{
    return MyMain::main();
}